### PR TITLE
chore(docs): add warning re: existing dirs named "copilot"

### PR DESCRIPTION
**Pull request by huanjani**
Please help with the wording of this! 🥺 
Addresses: #3294, #4681.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the Apache 2.0 License.

### CODE REVIEW

**Review by Lou1415926 — COMMENTED** [review]
I have two little thoughts to share 💭 

1. Looking at https://github.com/aws/copilot-cli/issues/4681, I wonder if searching up to 5 levels is too deep? Maybe we just need to search for 3 levels?
2. Instead of suggesting that users rename their existing `copilot` folder, I wonder if we should instead suggest that users create a `copilot` folder at the correct place, so that we won't look for the wrong `copilot` further deep down.

**Comment by huanjani** [comment]
> 1. Looking at #4681, I wonder if searching up to 5 levels is too deep? Maybe we just need to search for 3 levels?

It could be possibly backwards-incompatible to change it though, right?

> 2. Instead of suggesting that users rename their existing `copilot` folder, I wonder if we should instead suggest that users create a `copilot` folder at the correct place, so that we won't look for the wrong `copilot` further deep down.

Yeah, I felt a little uncomfortable asking people to rename their dir. I like your solution!

File: site/content/docs/concepts/applications.en.md [hunk 1]
```diff
@@ -10,6 +10,9 @@ Your application configuration (which services and environments belong to it) is
 
 ## Creating an App
 
+!!! Attention
+If you have an existing directory named `copilot/` in your local directory that was not created by AWS Copilot, please ,make an empty directory also named `copilot` closer to your working directory.
```
Comment (Lou1415926) [inline site/content/docs/concepts/applications.en.md]:
```suggestion
    If you have an existing directory named `copilot/` in your local directory that was not created by AWS Copilot, please make an empty directory also named `copilot` closer to your working directory.
```

Comment (Lou1415926) [reply site/content/docs/concepts/applications.en.md]: In addition, what do you think if we describe the potential problem first? Something like:
> If you have an existing `copilot/` directory that you created for other purposes, you might find Copilot creating files in that directory. If this happens, you can create an empty directory named `copilot/` closer to your working directory. Copilot will instead use this empty directory.

**Review by Lou1415926 — APPROVED** [review]

**Review by dannyrandall — APPROVED** [review]

**Pull request merged**